tutorials/tour/basics.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: tutorial
+title: Basics
+
+disqus: true
+
+tutorial: scala-tour
+num: 2
+next-page: unified-types
+previous-page: tour-of-scala
+---
+
+In this page, we will cover basics of Scala.
+
+## Variables
+
+You can define variables with `var` keyword.
+
+```
+var x = 1 + 1
+x += 1
+```
+
+Often, you want your variables to be immutable. You can use `val` keyword in that case.
```

Comment:
I was thinking of something more along the lines of:

> You can define values with the `val` keyword.
>
> ```
> val x = 1 + 1
> ```
>
> Values cannot be re-assigned and are immutable in that sense. If you do require a mutable variable, however, you can use the `var` keyword. However, `val` should be used by default since `var` generally indicates that it *will* be mutated down the line.
>
> ```
> var y = 1 + 2
> y += 4
> ```

Feel free to further clean up the wording, but really all I meant was that I think we should avoid hand-wavy words like "idiomatic", and when we introduce something, we should briefly describe what it is.